Makefile

```diff
@@ -36,6 +36,7 @@ PKG          := ./...
 TAGS         :=
 TESTS        := ".*"
 TESTTIMEOUT  := 15s
+CPUS         := 1
```

Comment:
`  -test.cpu="": comma-separated list of number of CPUs to use for each test`

what does that even mean?

`-test.cpu=1,2` will run everything first with `GOMAXPROCS=1`, then again with `GOMAXPROCS=2`?